src/properties-panel/components/Group.js

```diff
@@ -40,4 +79,17 @@ export default function Group(props) {
       }
     </div>
   </div>;
+}
+
+function DataMarker() {
+  return (
+    <div class="bio-properties-panel-dot">
+      <svg
+        aria-label="dirty" role="img" xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 100 100"
+      >
+        <circle fill="currentColor" cx="50" cy="50" r="50" />
```

Comment:
Would we not rather style the color of the circle explicitly via css? This is where I would look for the style of it.

Reply:
I simply re-use the dot from the Camunda Modeler, to avoid different styles of doing it.

https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/version-info/VersionInfo.js#L132

Reply:
Ah, now I got what you meant. I simply used the default font color here (`currentColor`). We could decide to state `black` in the selector, but since we don't have a fixed color palette yet, I'd keep it to the default and change it once we do the design polishing.

Reply:
Just a comment: the unread marker for versions and the data marker for the properties panel are not the same UI component. Although they look similar (equal shape but different colors) this may change as we learn more about how they are perceived.